modules/landing_zone_reader_config/main.tf

```diff
@@ -0,0 +1,51 @@
+resource "null_resource" "landing_zone_reader_config" {
+  triggers = {
+    providers  = "${md5(jsonencode(var.landing_zone_providers))}"
+    components = "${md5(jsonencode(var.landing_zone_components))}"
+  }
+
+  provisioner "local-exec" {
+    when    = "create"
+    command = "sh ${path.module}/../scripts/config.sh"
+
+    environment = {
+      ROOT_PATH  = "${var.root_path}"
+      PROVIDERS  = "${jsonencode(var.landing_zone_providers)}"
+      COMPONENTS = "${jsonencode(var.landing_zone_components)}"
+    }
+  }
+
+  provisioner "local-exec" {
+    when    = "destroy"
+    command = "python ${path.module}/../scripts/remove_config.py"
```

Comment:
I believe this should be `sh ${path.module}/scripts/remove_config.py` (instead of `sh ${path.module}/../scripts/remove_config.py`)